lib/util/index.coffee

```diff
@@ -187,7 +187,13 @@ convertExpandToObject = (expandOption) ->
 	else if isString(expandOption)
 		return "#{expandOption}": {}
 	else if isArray(expandOption)
-		return fromPairs(expandOption.map((key) -> [key, {}]))
+		return fromPairs(expandOption.map((value) ->
+			if isString(value)
+				return [value, {}]
+			else
+				key = Object.keys(value).shift()
```

Comment:
This gets the first key (with somewhat arbitrary ordering) - I think it should probably be _all_ keys